### PR TITLE
Update InstallNuKeeperDotnetTool.bat

### PR DESCRIPTION
**Pull request by jisperliu**
pack artifacts to root directory instead of NuKeeper/artifacts

### :thinking: Checklist before submitting

- [x] All projects build
- [x] Relevant documentation was updated

### CODE REVIEW

**Comment by MarcBruins** [comment]
Thanks

**Comment by MarcBruins** [comment]
argh travis is failing, @AnthonySteele  can we deprecate travis?

**Comment by AnthonySteele** [comment]
Can do, we're confident that Azure can do it. I will have a poke around to find where to remove the integration.

**Comment by AnthonySteele** [comment]
Gone!

**Comment by AnthonySteele** [comment]
Do we still want this one?

File: InstallNuKeeperDotnetTool.bat [hunk 1]
```diff
@@ -5,7 +5,7 @@ echo Build solution:
 dotnet build 
 
 echo Pack NuKeeper dotnet tool:
-dotnet pack .\NuKeeper\NuKeeper.csproj -o ".\artifacts"
+dotnet pack .\NuKeeper\NuKeeper.csproj -o "..\artifacts"
```
Comment (AnthonySteele) [inline InstallNuKeeperDotnetTool.bat:9]: Does line 14 need to change to match line 8?

**Pull request closed without merging**